libs/service-portal/settings/access-control/src/screens/Access/Access.treat.ts

```diff
@@ -28,3 +28,8 @@ export const closeButton = style({
   right: theme.spacing['1'],
   zIndex: 2,
 })
+
+export const table = style({
+  marginBottom: '200px',
```

Comment:
Adding some margin to reduce the UI flickering. The needed margin is 350px, but it didn't look good in regards of the footer.

Reply:
I'd like to find a better solution, as this one adds unnecessary margin and painful scroll issues.

Do you know why the table needs overflow hidden/scroll at all? If so, we probably need to implement a way to render the open datepicker modal outside the table or in a portal, with correct absolute position.

Reply:
Yeah I agree, from our conversation, this was supposed to be a temp solution (because of the summer vacations) until a better solution would be implemented:
```
þarf að vera leyst á meiri global hátt, eins og t.d. í mobile verði þetta modal og þannig. En fínt í bili
```

I didn't want to change the overflow since I didn't know why it was set in the first place and might affect other applications.
I'll remove the className and try to find a better solution.

Reply:
That's fair. Would be good to git blame and ask the component author sometime.

Reply:
I'd even prefer a "temporaryDropdownSupport" prop on Table. 💩

Reply:
Contacted the "author" and as far as he could remember, this was supposed to be for horizontal scrolling; so we can "safely" remove the overflow hidden.

Reply:
Ahhh makes sense. Horizontal scrolling is actually quite important for responsive tables, e.g. so we can fit this table on mobile. Not that it's ideal UX in any way.

But maybe we can cheat for now with a `disableHorizontalScrolling` prop? :/

Reply:
The table is wrapped in a div which has: `overflow: auto`. In addition to that, the table inside the div had `overflow: hidden` which I believe is not needed, thus we can remove that.

Reply:
https://github.com/island-is/island.is/pull/4456/files#diff-bb7eb0ed44e43426c91ec7e387240ed091adba21dd5cef938cd1558b6ec620e9R31